### ISSUE TITLE
Random weirdness with SolderJumpers

### ISSUE THREAD

**Issue by cdntinker** [easyw/kicadStepUpMod#191]
Something very strange keeps happening with SolderJumpers.
They sometimes seem to import as bizarre shapes.
This seems to happen randomly.  Doesn't seem entirely consistent. Some work, some don't.
Hard to describe, so I'll try to attach some images.

Linux (LMDE5)
FreeCAD (0.19)
kicadStepUp (10.20.7)
![hub board](https://github.com/easyw/kicadStepUpMod/assets/58090022/2f63d8fc-e0dc-491f-ab01-4111797135db)
![test board top](https://github.com/easyw/kicadStepUpMod/assets/58090022/d8122976-5cf3-4556-b4b3-f2077f840941)
![test board bottom](https://github.com/easyw/kicadStepUpMod/assets/58090022/db0a7a79-b077-46a3-933d-286582622fef)

**Comment by easyw** [comment]
@cdntinker 
probably this is related tp the jumper footprint content. Not all the pad shapes are supported ATM.
Do you have a sample board to check?

**Comment by cdntinker** [comment]
Apparently, GitHub doesn't like kicad_pcb files...
So I zipped it.
This is the board file that created the 2nd & 3rd pics above.

[zz.kicad_pcb.zip](https://github.com/easyw/kicadStepUpMod/files/13191856/zz.kicad_pcb.zip)

**Comment by luzpaz** [comment]
@cdntinker you're using an obsolete version of freecad. Current stable is 0.21.1

**Comment by cdntinker** [comment]
@luzpaz 
Yes, I'm running the version that works with the current version of my OS.
One day, I'll take the time & find the energy to fully upgrade my dev machine from LMDE 5 to LMDE 6 & find out what versions of things work & break.

**Comment by luzpaz** [comment]
@easyw can you reproduce with the attached file ?

**Comment by easyw** [comment]
this is related to the net tie fp and FreeCAD... the net-tie-fp geometry goes into coplanar OCC issues
https://discourse.mcneel.com/t/coplanar-surfaces-boolean-vs-planar-for-union-difference-and-intersect/152291
Unfortunately not all the kicad pad type can be fully converted to a clean geometry.
The OP may consider to move to a different net tie footprint as suggested here:
https://github.com/easyw/kicadStepUpMod/issues/196#issuecomment-1807756868
